src/components/header/header.spec.js

```diff
@@ -0,0 +1,116 @@
+import { renderComponent, setTestPage } from '../../tests/helpers/rendering';
+import { EXAMPLE_HEADER_SEARCH_AND_MENU_LINKS } from './_test-examples';
+import { getNodeAttributes } from '../../tests/helpers/puppeteer';
+
+describe('script: header', () => {
```

Comment:
```suggestion
describe('FOR: Script: Header', () => {
```

Reply:
we are not using this format for script tests